Sources/XiEditor/AppDelegate.swift

```diff
@@ -98,29 +98,11 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }
 
     lazy var defaultConfigDirectory: URL = {
-        let applicationDirectory = FileManager.default.urls(
-            for: .applicationSupportDirectory,
-            in: .userDomainMask)
-            .first!
+        return FileManager.default.urls(        
```

Comment:
can we clean up the trailing whitespace? there's a good setting in Xcode for this:
![Screen Shot 2019-08-06 at 11 40 47 AM](https://user-images.githubusercontent.com/3330916/62554569-529d4600-b83f-11e9-88f4-37711005604a.png)